include/kllvm/ast/AST.h

```diff
@@ -1023,9 +1023,9 @@ class kore_definition {
   [[nodiscard]] std::list<kore_axiom_declaration *> const &get_axioms() const {
     return axioms_;
   }
-  [[nodiscard]] kore_axiom_declaration *
+  [[nodiscard]] kore_axiom_declaration
```

Comment:
We don't want to copy this out, but rather return a const reference to the object that's stored.